pkgs/by-name/yo/yopass/package.nix

```diff
@@ -0,0 +1,50 @@
+{
+  buildGoModule,
+  callPackage,
+  fetchFromGitHub,
+  makeBinaryWrapper,
+  lib,
+  nix-update-script,
+}:
+buildGoModule (finalAttrs: rec {
+  pname = "yopass";
+  version = "12.1.0";
+
+  src = fetchFromGitHub {
+    owner = "jhaals";
+    repo = "yopass";
+    tag = "${version}";
+    hash = "sha256-5iUiVASrvzpqnnCyV1mB8psF/ERaazc2/1riBeVIDag=";
+  };
+
+  vendorHash = "sha256-5mC5l+ajnmyN+fbxuc/ub9QoKcF3T4xxPBu8H/RTncA=";
+
+  passthru = {
+    inherit version;
+    updateScript = nix-update-script { };
```

Comment:
If you define an update script that script should also be able to update the hash for yarnOfflineCache.